### PR TITLE
[SB-CD] Add Icons 

### PR DESCRIPTION
**Pull request by sethonne**
## Describe your changes here
- Added icons to pokemon details page

## Issue ticket number and link
What issue does this PR reference?
[Ticket-0](https://trello.com/c/vGxiKhBX/9-add-icons)

## Screenshots/Images
image

### CODE REVIEW

**Comment by Luzefiru** [comment]
I added colors & icons to the types as well.

image

**Review by 0-BSCode — APPROVED** [review]

**Comment by 0-BSCode** [comment]
Don't merge yet. There're icons for the filter modal that need to be placed after the search and sort feature is merged.

**Comment by Luzefiru** [comment]
- Added an icon to open the search & filtering modal.
- Added padding to the modal to prevent the sorting buttons from overflowing on web.

## Screenshot
Screenshot_1716365093

**Review by 0-BSCode — APPROVED** [review]

**Pull request merged**
